14/umbraco-cms/extending/dashboards.md

```diff
@@ -23,26 +49,95 @@ This section dives into the Dashboard Extension Manifest, shows how to register
 
 ### Example Extension Manifest
 
-Insert this as an entry of extensions in your `Umbraco-package.json` or register it via JavaScript.
+{% hint style="info" %}
+You can read more about manifests in the tutorial [Creating Your First Extension](../tutorials/creating-your-first-extension.md).
+{% endhint %}
+
+Insert this as an entry in the `extensions` list in a `umbraco-package.json` file.
 
-```jsx
+{% code title="~/App_Plugins/WelcomeDashboard/umbraco-package.json" lineNumbers="true" %}
 
+```json
 {
- "type": "dashboard",
- "alias": "my.welcome.dashboard",
- "name": "My Welcome Dashboard",
- "element": "/App_Plugins/WelcomeDashboard/dashboard.js",
- "elementName": "my-welcome-dashboard",
- "weight": -1,
- "meta": {
-  "label": "Welcome Dashboard",
-  "pathname": "welcome-dashboard"
- },
- "conditions": [
+  "type": "dashboard",
+  "alias": "my.welcome.dashboard",
+  "name": "My Welcome Dashboard",
+  "element": "/App_Plugins/WelcomeDashboard/dashboard.js",
+  "weight": -1,
+  "meta": {
+    "label": "Welcome Dashboard",
+    "pathname": "welcome-dashboard"
+  }
+}
+```
+
+{% endcode %}
+
+This will register a dashboard with the alias `my.welcome.dashboard` and the name `My Welcome Dashboard`. The dashboard will be loaded from the file `/App_Plugins/WelcomeDashboard/dashboard.js`. The dashboard will be displayed with the label `Welcome Dashboard` and the URL `/welcome-dashboard` on *all sections*, e.g. `/section/content/dashboard/welcome-dashboard`.
+
+### Conditions
+
+You can specify conditions for when the dashboard should be displayed. This is done by adding a `conditions` property to the manifest. Ideally we would like the dashboard to be shown only on a specific section. This can be done by specifying the condition called `Umb.Condition.SectionAlias` and providing the [alias of the section](#default-dashboards-in-umbraco) you want the dashboard to be displayed on:
```

Comment:
```suggestion
You can specify conditions for when the dashboard should be displayed. This is done by adding a `conditions` property to the manifest. Ideally, we would like the dashboard to be shown only in a specific section. This can be done by specifying the condition called `Umb.Condition.SectionAlias` and providing the [alias of the section](#default-dashboards-in-umbraco) you want the dashboard to be displayed on:
```